StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentSheet+API.swift

```diff
@@ -390,7 +395,7 @@ extension PaymentSheet {
             }
         }
 
-        let options = STPConfirmPaymentMethodOptions()
+        let options = params.paymentMethodOptions ?? STPConfirmPaymentMethodOptions()
```

Comment:
Given that we initialise this variable in constructor do we need nil-coalescing here?

Reply:
I think this is necessary as `paymentMethodOptions` [is defined as optional](https://github.com/stripe/stripe-ios/blob/c2bb1fb7c8795f3e3754b81c3258d7791a204c02/StripePayments/StripePayments/Source/API%20Bindings/Models/PaymentIntents/STPPaymentIntentParams.swift#L148C75-L149C1) . Also I believe it could be nil if the `.saved` case in the switch block above it is used.